assets/js/modules/thank-with-google/components/setup/SetupMain.js

```diff
@@ -1,5 +1,5 @@
 /**
```

Comment:
@kuasha420 Within the same file, the module title should be updated from `Subscribe with Google` to `Thank with Google`